benchmark/single-source/DataBenchmarks.swift

```diff
@@ -397,12 +410,19 @@ public func string(_ N: Int, from data: Data) {
 }
 
 @inline(never)
-public func data(_ N: Int, from string: String) {
+public func dataFromUTF8View(_ N: Int, from string: String) {
   for _ in 1...N {
     blackHole(Data(string.utf8))
   }
 }
 
+@inline(never)
+public func dataUsingUTF8Encoding(_ N: Int, from string: String) {
+  for _ in 1...N {
+    blackHole(string.data(using: .utf8))
```

Comment:
It looks like we are leaking Obj-C objects here… I think we need to do this:
```suggestion
    autoreleasepool { blackHole(string.data(using: .utf8)) }
```